standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/properties/Digester.java

```diff
@@ -17,6 +17,8 @@
  */
 package org.apache.hadoop.hive.metastore.properties;
 
+import org.jetbrains.annotations.NotNull;
```

Comment:
nit: looks like this import is none of use.

Reply:
Slipped through the cracks, thanks.